slicer-test/opengl/CircleCI_Slicer_Docker.cmake

```diff
@@ -84,7 +84,15 @@ ctest_start( "${SITE_CTEST_MODE}" )
 #ctest_build( BUILD ${CTEST_BINARY_DIR} )
 #    TARGET "${TEST_GUI}" )
 
-ctest_test( BUILD "${CTEST_BINARY_DIRECTORY}" )
+ctest_test( BUILD "${CTEST_BINARY_DIRECTORY}"
+    RETURN_VALUE ctest_return_value )
 #    INCLUDE "${RUN_TEST_GUI}" )
 
+message( "\n ctest_return_value: ${ctest_return_value}\n" )
+
 ctest_submit()
+
+if(NOT ctest_return_value EQUAL 0)
+  message( "\n" )
```

Comment:
Still not beautiful I think
![image](https://cloud.githubusercontent.com/assets/20341270/21068890/f210d110-be42-11e6-80df-123ac176d411.png)